examples/tree_2d_dgsem/elixir_eulermulti_shock_bubble_shockcapturing_subcell_positivity.jl

```diff
@@ -133,10 +133,9 @@ callbacks = CallbackSet(summary_callback,
 ###############################################################################
 # run the simulation
 
-output_directory = "out"
 stage_callbacks = (SubcellLimiterIDPCorrection(),
-                   BoundsCheckCallback(save_errors = true, interval = 100,
-                                       output_directory = output_directory))
+                   BoundsCheckCallback(save_errors = false, interval = 100,
+                                       output_directory = "out"))
```

Comment:
I don't understand - why do I have to specify an output directory even if I do not want to `save_errors`?

Reply:
I still want to test output in this elixir. I thought it is cleaner to keep this disabled in the elixir but activate it in the tests with
```
    @test_trixi_include(joinpath(EXAMPLES_DIR,
                                 "elixir_euler_sedov_blast_wave_sc_subcell.jl"),
                        l2=[
                            0.4328635350273501,
                            0.15011135840723572,
                            0.15011135840723572,
                            0.616129927549474,
                        ],
                        linf=[
                            1.6145297181778906,
                            0.8614006163026988,
                            0.8614006163026972,
                            6.450225090647602,
                        ],
                        tspan=(0.0, 1.0),
                        initial_refinement_level=4,
                        coverage_override=(maxiters = 6,),
                        save_errors=true,
                        output_directory="out")
```

To specify the parameter `save_errors` and `output_directory` from the test, they need to be written out in the elixir.
Alternatively, I could rely on the fact that the default output directory isn't change and always just take `"out"` for the test without further specification. Then, if someone changed the directory manually in the elixir, the test would fail.

Reply:
I now removed the `output_directory="out"` from the elixirs and the tests setups. The tests now use the default directory "out".
I kept `interval=100` in the elixirs, to not generate a too large output file during the tests.